hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/HoodieLogFileCommand.java

```diff
@@ -92,6 +92,9 @@ public String showLogFileCommits(
         if (n instanceof HoodieCorruptBlock) {
           try {
             instantTime = n.getLogBlockHeader().get(HeaderMetadataType.INSTANT_TIME);
+            if (instantTime == null) {
+              throw new Exception("Invalid value " + instantTime + " for instant time ");
```

Comment:
change to  : "Invalid instant time" + null

Reply:
done